src/App.tsx

```diff
@@ -37,7 +36,7 @@ function mapDispatchToProps(dispatch) {
  * @description - Root level component for VoTT Application
  */
 @connect(mapStateToProps, mapDispatchToProps)
-class App extends React.Component<IAppProps> {
+export default class App extends React.Component<IAppProps> {
```

Comment:
This component was exported weirdly - made consistent with other page components.